src/typings/che.d.ts

```diff
@@ -29,10 +29,13 @@ declare namespace che {
     usedResources?: string;
   }
 
+  export type WorkspaceStorageType = 'async' | 'ephemeral' | 'persistent';
```

Comment:
I prefer to use an enum.  Just try to imagine how many places should be changed in the case of renaming

Reply:
I've already refactored related code. 
If you want let's continue discussion [here](https://github.com/che-incubator/che-dashboard-next/pull/83#pullrequestreview-537736814).

Reply:
ok